vendor/vendor.json

```diff
@@ -2387,7 +2387,7 @@
 			"revision": "604eaf189ee867d8c147fafc28def2394e878d25"
 		},
 		{
-			"checksumSHA1": "rFSw9cpg4vxtZD7dlOtKx25h4kI=",
+			"checksumSHA1": "2jmj7l5rSw0yVb/vlWAYkK/YBwk=",
```

Comment:
With the latest version of govendor, I get a `checksumSHA1` of `rFSw9cpg4vxtZD7dlOtKx25h4kI=`, not `2jmj7l5rSw0yVb/vlWAYkK/YBwk=`

Reply:
done